cmd/create/accountroles/cmd.go

```diff
@@ -363,7 +363,16 @@ func createRoles(reporter *rprtr.Object, awsClient aws.Client,
 		filename = fmt.Sprintf("sts_%s_permission_policy.json", file)
 		path = fmt.Sprintf("templates/policies/%s", filename)
 
-		policy, err = aws.ReadPolicyDocument(path)
+		// For the installer permission policy scope the iam:PassRole permission
+		// to the controlplane ARN role only.
+		// The cluster itslef will provision worker nodes see machine-api-operator
+		// permission.
+		if strings.ToLower(role.Name) == aws.InstallerAccountRole {
```

Comment:
Remove the conditional, otherwise only the installer will have a policy :P

Reply:
:facepalm: